packages/blocks/src/components/form-checkbox/index.js

```diff
@@ -11,7 +11,7 @@ import { popAnimation } from '@crowdsignal/styles';
 
 const CheckboxLabel = styled.label`
 	display: flex;
-	align-items: baseline;
+	align-items: center;
```

Comment:
Have we checked how these changes affect multiple choice question blocks? If they don't break, it's a welcome improvement, otherwise we probably should probably leave this as you've done and move these styles to somewhere inside the multiple choice answer block.

Reply:
No regression on multiple choice blocks.